client/src/components/NewsArticle/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './style.scss';
+import teamImg from '../../assets/img/Basketball.png';
+
+export const NewsArticle = () => (
+  <section className='single-news-container'>
+    <div className='news-wrapper'>
+      <div className='top-info'>
+        <h4>
+          <span className='team-name'>NBA </span> <p className='team-paragraph'> &gt; Tennessee / As M.L.B. &apos; s Season Opens in Japan, at Least the Dirt Is Familiar</p>
+        </h4>
+      </div>
+      <div className='image-and-card-container'>
```

Comment:
Can be not only image and text. So bad class name

Reply:
renamed